SimplePagedViewFramework/Source/SimplePagedView.swift

```diff
@@ -1,19 +1,23 @@
 import UIKit
 
-public class SimplePagedView: UIViewController {
+public class SimplePagedView: UIView {
+
+    public enum ScrollDirection {
```

Comment:
I think we can get rid of this. Might be nice to have at some point, but no need for this release